### PR TITLE
Inventory controller

### PR DESCRIPTION
**Pull request by rhys-saldanha**
and some tidying

### CODE REVIEW

**Comment by rhys-saldanha** [comment]
Probably needs more tests

**Review by VikashKothary — DISMISSED** [review]
LGTM. Just need to handle the merge conflicts. Will the codecov prevent merging unless code coverage is 100%?

**Comment by rhys-saldanha** [comment]
LGTM? Yeah it should prevent the merge, which is good, I should and will add tests

**Review by VikashKothary — APPROVED** [review]

File: web/src/main/java/org/reciplease/configuration/LocalDataProducer.java [hunk 1]
```diff
@@ -2,24 +2,29 @@
...
 @Profile("local")
 @RequiredArgsConstructor
 public class LocalDataProducer implements ApplicationRunner {
     final IngredientRepository ingredientRepository;
+    final InventoryRepository inventoryRepository;
 
     @Override
     public void run(final ApplicationArguments args) {
-        final List<Ingredient> ingredients = List.of(
+        final var ingredients = ingredientRepository.saveAll(List.of(
```
Comment (VikashKothary) [inline web/src/main/java/org/reciplease/configuration/LocalDataProducer.java:27]: Oooh. Controversial :P

**Review by VikashKothary — APPROVED** [review]

**Pull request closed without merging**